doc/Analytics.xml

```diff
@@ -1821,7 +1821,7 @@
                     <para>fc: AccessoryDescription</para>
                   </entry>
                   <entry>
-                    <para>N/A</para>
+                    <para>fc:HelmetType</para>
```

Comment:
**Suggestion :** "fc:subtype" includes "fc:HelmetType" string
- just to make sure the documentation provides reference to new "subtype" element.

Reply:
I agree with @bsriramprasad suggestion.